src/components/Sections/SectionNovelty/index.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <section class="novelty">
+        <div class="novelty__box container">
+            <img src="../../../assets/new-coffee.png" alt="new-coffee picture"
+            class="novelty__picture">
+            <div class="novelty__describe">
+                <p class="novelty__before-title">What’s new?</p>
+                <h2 class="novelty__title">Meet our new coffee!</h2>
+                <p>Try one of our signature selections and see what everyone’s
+                talking about or select ‘Catalogue’ at the top of the
+                page to view all the possibilities.</p>
+                <a href="" class="novelty__click link">Learn more</a>
+            </div>
+        </div>
+    </section>
+</template>
+
+<script>
+export default {
+  name: 'SectionNovelty',
```

Comment:
If you use the name SectionNovelty for a component, it is expected that the component's class will be named "section-novelty"